openpype/hosts/maya/plugins/publish/collect_render.py

```diff
@@ -234,13 +234,14 @@ def process(self, context):
             publish_meta_path = None
             for aov in exp_files:
                 full_paths = []
-                for file in aov[aov.keys()[0]]:
+                aov_first_key = list(aov.keys())[0]
+                for file in aov[aov_first_key]:
```

Comment:
Py3 `dict_keys` are not subscriptable (can't index them directly) and this avoids that error whilst being backwards compatible with Py2